css/main.css

```diff
@@ -357,8 +358,25 @@ table th {
 		padding-bottom:0;
 	}
 	
-	#menu { 
-		display:none; 
+	#menu, 
+	#menu * { 
+		display:none!important;
+		visibility: hidden;
 	}
 	
+	.section .mod1, .section .mod1 h1, .mod1 { background-color:#000; color:#FFFFFF; }
```

Comment:
Can you not implement all of these with ".section span { background-color: #000 !important; color: #FFFFFF !important; }"?